### ISSUE TITLE
Mentions in the activity center are not shown for 1:1 chats

### ISSUE THREAD

**Issue by Michal-Status** [status-im/status-desktop#13524]
# Bug Report

## Description

When a contact mentions us in communities or group chat, the mention is displayed in the activity center alongside any message and images that were sent with that mention. This does not work when a user mentions another user in a 1:1 chat.

## Steps to reproduce

1. Log into two users that have each other in their contacts
2. As User2 mention User1 in a 1:1 conversation

#### Expected behavior
Mention is displayed in the activity center

#### Actual behavior
Mention is not displayed in the activity center

### Additional Information

- Status desktop version: https://ci.infra.status.im/job/status-desktop/job/release/job/release%2F2.27.x/6/
- Operating System: macOS

https://github.com/status-im/status-desktop/assets/155960867/e3903a1e-b136-4ca1-b127-527fd6692673

**Comment by jrainville** [comment]
I think this is more a design decision. There is no actual reason to mention someone in a 1 on 1, since those are direct messages.

Do you think we should implement it @benjthayer ?

**Comment by jrainville** [comment]
I'm taking the executive decision to close this as I don,t see the point. Feel free to reopen if you think it's worth it.

**Comment by benjthayer** [comment]
> I'm taking the executive decision to close this as I don't see the point. Feel free to reopen if you think it's worth it.

Yeah I don't think we need notification centre items for 1:1 chats. NC items are useful for mentions in group chats and channels because they allow the user to navigate directly to the relevant message amongst potentially 100's of other messages. But in a 1:1 chat, every message technically is a mention and the user can just go to the chat to read any unread messages.
